skin.json

```diff
@@ -137,6 +137,10 @@
     "timeToShow": "10"
   },
   "controlBar": {
+    "volumeView": {
```

Comment:
View is kind of a loaded term, can we go with volumeControl or volumeBar or something along those lines.